src/tools/ebuspicloader.cpp

```diff
@@ -252,7 +252,11 @@ typedef union
 #define FRAME_HEADER_LEN 9
 #define FRAME_MAX_LEN (FRAME_HEADER_LEN+2*WRITE_FLASH_BLOCKSIZE)
 #define BAUDRATE_LOW B115200
+#if !defined(__APPLE__)
 #define BAUDRATE_HIGH B921600
+#else
+#define BAUDRATE_HIGH 230400
```

Comment:
why is this needed?

Reply:
The MacSDK's termios.h in /Library/Developer/CommandLineTools/SDKs/MacOSX13.3.sdk/System/Library/Frameworks/Kernel.framework/Versions/A/Headers/sys/termios.h shows these definitions at line 316:

```
#endif

/*
 * Standard speeds
 */
#define B0      0
#define B50     50
#define B75     75
#define B110    110
#define B134    134
#define B150    150
#define B200    200
#define B300    300
#define B600    600
#define B1200   1200
#define B1800   1800
#define B2400   2400
#define B4800   4800
#define B9600   9600
#define B19200  19200
#define B38400  38400
#if !defined(_POSIX_C_SOURCE) || defined(_DARWIN_C_SOURCE)
#define B7200   7200
#define B14400  14400
#define B28800  28800
#define B57600  57600
#define B76800  76800
#define B115200 115200
#define B230400 230400
#define EXTA    19200
#define EXTB    38400
#endif  /* (!_POSIX_C_SOURCE || _DARWIN_C_SOURCE) */


#if !defined(_POSIX_C_SOURCE) || defined(_DARWIN_C_SOURCE)
```
I assumed the highest baudrate reliably supported is 230400.

Reply:
AFAIR on Darwin you have to use IOSSIOSPEED to set non-standard speed and it works at least on FTDI for the `921600`